README.md

```diff
@@ -1,21 +1,42 @@
-# Paranomax Mobile App
+<p align="center">
+<img src="/assets/logo-mobile-app-beta.png" height="100"/>
+</p>
+<h1 align="center">Panoramax Mobile App</h1>
+
+Panoramax mobile is an application to capture photo sequences with your mobile and send these sequences to a Panoramax instance.
+
+> [!NOTE]
+> The app is under development, a first version is available [here](https://github.com/nobelization/panoramax-mobile-app/releases).
+> You can follow progression in this [project](https://github.com/orgs/nobelization/projects/1).
+
+<div align="center">

```

Comment:
Shouldn't we only have the APK link for the moment?

the others are not yet available, are they (I have not followed the latest updates on this topic)